src/desktop/components/split_test/test/split_test.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-done-callback */
```

Comment:
Happy to convert the `beforeEach` below to a Promise if you can offer a pointer or example.

Reply:
The `no done` rule is a bit extreme IMO /shrug. All good.